acceptance/acceptance_test.go

```diff
@@ -178,6 +187,86 @@ func thereShouldBeNoWarningsInTheResult(ctx context.Context) error {
 	return nil
 }
 
+func opaEval(ctx context.Context, evalString string) (context.Context, error) {
+	ts, err := getTestState(ctx)
+	if err != nil {
+		return ctx, fmt.Errorf("opaEval get test state: %w", err)
+	}
+
+	cmd := exec.Command(
+		"go",
+		"run",
+		"github.com/enterprise-contract/ec-cli",
+		"opa",
+		"eval",
```

Comment:
An argument for not using `opa eval` is that it doesn't represent what actually happens when the EC CLI runs, which is the main (only?) use for the policies in this repository.

But looking at the newly added feature file, I can definitely see how this can streamline things. I do wonder if that is better done as a unit test that uses real data :thinking: